packages/graphql/src/contracts.js

```diff
@@ -138,7 +138,10 @@ export function setNetwork(net, customConfig) {
   }
   clearInterval(blockInterval)
 
-  web3 = applyWeb3Hack(new Web3(config.provider))
+  const provider = process.env.WEB3_PROVIDER_URL
```

Comment:
I think we use PROVIDER_URL for other services. So let's be consistent. plus it is what got added in README above :)